components/resources/brave_new_tab_page_strings.grdp

```diff
@@ -173,8 +173,12 @@
     Show search widget in new tabs
   </message>
 
+  <message name="IDS_NEW_TAB_SHOW_SPONSORED_IMAGES_EARNING_TEXT" desc="Text describing earning for new tab page ads">
+    You can earn for seeing new tab page ads with <ph name="LINK_BEGIN">$1</ph>Brave Rewards<ph name="LINK_END">$2</ph>.
```

Comment:
This sounds a bit odd to me, maybe flip it like `With Brave Rewards you can earn for seeing new tab page ads`, but also it's omitting what you would earn.

Reply:
@Miyayes WDYT?

Reply:
Yes, let's flip to: 

With Brave Rewards, you can earn for seeing new tab page ads.